vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/Slide.java

```diff
@@ -28,11 +32,16 @@
         strategy = "edu.asu.diging.vspace.core.data.IdGenerator")
     private String id;
 
-    @OneToOne(targetEntity = Module.class)
+    @ManyToOne(targetEntity = Module.class)
     private IModule module;
 
+    @JsonIgnore
     @OneToMany(targetEntity = ContentBlock.class, mappedBy = "slide", cascade = CascadeType.ALL)
     private List<IContentBlock> contents;
+    
+    @JsonIgnore
```

Comment:
why the ```JsonIgnore```?

Reply:
I was getting a Jackson databind error without the JsonIgnore annotation as there is a circular reference.

Reply:
are you returning the slide in the controller at some point? is that why you need the jackson annotation? if so, please make a comment.

Reply:
a comment in the code I mean ;)

Reply:
Oh yeah Sure. I also feel it could be achieved in another way without ResponseEntity somehow I could not achieve that. Could you also review that part?

Reply:
You could either create a new slide object to be returned by the controller in a responseentity that has only the properties you need in the frontend, or build the json directly yourself (same thing, only include the properties you need). both approaches would be cleaner (model wouldn't be mixed with frontend (jsonignore) code), but more coding.